Added basic tutorial

twinblade02 opened the pull request:
basic tutorial on reading and plotting with geopandas, some mention of other tools such as Folium.

Requesting code review.

twinblade02 committed:
Added basic tutorial

basic tutorial on reading and plotting with geopandas, some mention of other tools such as Folium.

Requesting code review.

harshpavuluri reviewed (changes requested):
Notebook looks good in terms of code. I'd say the only things that need to get cleaned up is the markdown sections. Make sure to use headers and such to make sure that it looks presentable. Some of my previous examples should be there for you to reference on how to format that section

twinblade02 committed:
Update BasicGeopandasTutorial.ipynb

Added headings and cleared up markdown a bit. Removed paths, replaced with placeholder.

harshpavuluri reviewed (approved):
LGTM
